samples/aspnetcore/SwaggerSample/ImplicitApiVersionParameter.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace Microsoft.Examples
+{
+    using Microsoft.AspNetCore.Mvc.ApiExplorer;
+    using Swashbuckle.AspNetCore.Swagger;
+    using Swashbuckle.AspNetCore.SwaggerGen;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Represents the Swagger/Swashbuckle operation filter used to document the implicit API version parameter.
+    /// </summary>
+    public class ImplicitApiVersionParameter : IOperationFilter
```

Comment:
I guess my comments weren't clear enough, but this particular filter handles both query string and URL path segment. Most service authors need one or the other. I wanted to show both working without creating another sample application.

In the URL path segment scenario, the only thing you need to do is assign the **Default** property.  This will fill in the current version API so that you can use the _Try It!_ feature. In the query string scenario, you have to add an implicit parameter that isn't formally declared in your controller actions. This would also be the case for versioning by header.